mgnify_pipelines_toolkit/analysis/amplicon/make_asv_count_table.py

```diff
@@ -43,10 +43,13 @@ def parse_args():
 
 def order_df(taxa_df):
 
-    if len(taxa_df.columns) == 7:
-        taxa_df = taxa_df.sort_values(["Kingdom", "Phylum", "Class", "Order", "Family", "Genus"], ascending=True)
```

Comment:
The only thing that caught my eye is names "Kingdom", "Phylum", "Class",... in the body of script. Ideally, it should go as global variable (even Bacteria and Archaea) or global dictionary. 
We have some examples of using "duplicated meaningless" globals like [here](https://github.com/EBI-Metagenomics/mi-automation/blob/develop/backlog/models.py#L793). But if taxonomy community wants to rename **Archaea** to **_Archea_** or **_arcaea_** it would be a pain to rename it in whole script.
That is just comment. Don't insist on fixing it now, just think :)

Reply:
I agree, this makes sense! We have a folder for constants, I'm gonna add them there in a new script, thank you 😄